connectors/v2/spotify-open.js

```diff
@@ -9,16 +9,16 @@
 Connector.playerSelector = '.now-playing-bar';
 
 Connector.getArtist = function() {
-	let artists = $('.now-playing-bar span > span > a').toArray();
+	let artists = $('.track-info__artists a').toArray();
 	return Util.joinArtists(artists);
 };
 
-Connector.trackSelector = '.now-playing-bar div > div > div > a';
+Connector.trackSelector = '.track-info__name a';
 
-Connector.trackArtImageSelector = '.now-playing-bar .cover-art-image-loaded';
+Connector.trackArtImageSelector = '.now-playing__cover-art .cover-art-image-loaded';
 
-Connector.isPlaying = function() {
-	return $('.control-button[title="Pause"]').length > 0;
-};
+Connector.playButtonSelector = '.control-button[class^="spoticon-pause-"]';
```

Comment:
This change makes the connector think the song is playing while it is actually paused and vice versa. Please use `spoticon-play-*` selector to avoid this behaviour.